frontend/src/stores/hydrologic.js

```diff
@@ -168,17 +169,21 @@ export const useHydrologicStore = defineStore('hydrologic', () => {
       fileType: 'all',
       default: true,
       short_definition: 'Longitude',
+      swotviz_alias: 'Longitude',
       units: 'decimal degrees',
-      plottable: false
+      plottable: false,
+      significant_figures: 4
     },
     {
       abbreviation: 'y',
       definition: 'Latitude of the node or reach ranging from 90°S to 90°N',
       fileType: 'all',
       default: true,
       short_definition: 'Latitude',
+      swotviz_alias: 'Latitude',
       units: 'decimal degrees',
-      plottable: false
+      plottable: false,
+      significant_figures: 4
```

Comment:
appears lots of your metadata updates from https://github.com/CUAHSI/SWOT-Data-Viewer/pull/95 included here in this PR...